packages/typespec-client-generator-core/src/internal-utils.ts

```diff
@@ -261,7 +260,7 @@ export function getSdkTypeBaseHelper<TKind>(
   return diagnostics.wrap({
     __raw: type,
     deprecation: getDeprecationDetails(context.program, type)?.message,
-    kind,
+    kind: kind,
```

Comment:
nit, the same as but simpler
```suggestion
    kind,
```